marge/single_merge_job.py

```diff
@@ -61,7 +62,17 @@ def update_merge_request_and_accept(self, approvals):
                 merge_request.comment("Someone skipped the queue! Will have to try again...")
                 continue
 
-            log.info('Commit id to merge %r (into: %r)', actual_sha, target_sha)
+            if _updated_sha == actual_sha and self._options.guarantee_final_pipeline:
+                log.info('No commits on target branch to fuse, triggering pipeline...')
+                merge_request.comment("jenkins retry")
```

Comment:
How does this guarantee that the final pipeline is run? It seems especially suspicious to add a comment saying "jenkins retry" here. Is that some internal smarkets CI stuff that's leaking out here? :)

Reply:
shouldn't this be a merge_request_pipeline_trigger call instead ? Maybe I'm missing something ...

Reply:
This is essentially what was done in the community fork in this MR: https://gitlab.com/marge-org/marge-bot/-/merge_requests/387. 

This upstream project is no longer maintained as far as we can tell.